mobile/src/main/java/org/openhab/habdroid/ui/OpenHABMainActivity.java

```diff
@@ -1140,6 +1066,85 @@ private void hideSnackbar() {
         }
     }
 
+    private void handleHttpFailure(Request request, int statusCode, Throwable error) {
```

Comment:
Please consider changing the name to `handleServerCommunicationFailure`, because it also handles non-http errors like SSL errors.

Reply:
Done.